### PR TITLE
BZ2025588: Fix the procedure for obtaining hub cluster deployment CRs

### PR DESCRIPTION
**Pull request by ogradyp**
Applies only to the Enterprise-4.9 branch & main

Browse the Bugzilla here:
https://bugzilla.redhat.com/show_bug.cgi?id=2025588

Browse the preview update here:
https://deploy-preview-39954--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected

### CODE REVIEW

File: modules/ztp-preparing-the-hub-cluster-for-ztp.adoc [hunk 1]
```diff
@@ -19,7 +19,15 @@ $ oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data
...
-.. Clone the Git repository.
+.. Extract the ArgoCD deployment CRs from the ZTP site generator container using the latest container image version:
++
+[source,terminal]
+----
+$ mkdir ztp
+$ podman run --rm -v `pwd`/ztp:/mnt/ztp:Z registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.9.0-1 /bin/bash -c "cp -ar /usr/src/hook/ztp/* /mnt/ztp/"
+----
++
+The remaining steps in this section relate to the `ztp/gitops-subscriptions/argocd/` directory.
```
Comment (michaelryanpeter) [inline modules/ztp-preparing-the-hub-cluster-for-ztp.adoc:30]: > The remaining steps in this section relate to the `ztp/gitops-subscriptions/argocd/` directory.

I am getting tripped up due to the context as part of the `3a` task.

Does it make sense to change this into a substep (3b) and have the customer navigate to the `ztp/gitop-subscriptions/argocd/` directory?

Or simply add the full file path to the files below?

File: modules/ztp-preparing-the-hub-cluster-for-ztp.adoc [hunk 2]
```diff
@@ -89,7 +97,7 @@ spec:
        syncOptions:
        - CreateNamespace=true
 ----
-<1> `path` is the branch that contains the `policygentemplates` CRs for the clusters.
+<1> `path` is the file path that contains the `policygentemplates` CRs for the clusters.
 <2> `repoURL` is the URL of the Git repository that contains the `policygentemplates` custom resources that specify configuration data for the site.
 <3> `targetRevision` is the branch on the Git repository that contains the relevant configuration data.
```
Comment (michaelryanpeter) [inline modules/ztp-preparing-the-hub-cluster-for-ztp.adoc:103]: While you are here, could you please add a `$` to `oc apply -k ./deployment`?

**Review by michaelryanpeter — COMMENTED** [review]
I left two small comments. Please reach out if you have questions.

File: modules/ztp-preparing-the-hub-cluster-for-ztp.adoc (hunk 1 above)
Comment (ogradyp) [reply modules/ztp-preparing-the-hub-cluster-for-ztp.adoc:30]: Thanks Michael. I think your second suggestion is the clearer option.
i will change the instances as follows: 
<1> `path` is the ztp/gitops-subscriptions/argocd/ file path that contains the `policygentemplates` CRs for the clusters.

File: modules/ztp-preparing-the-hub-cluster-for-ztp.adoc (hunk 2 above)
Comment (ogradyp) [reply modules/ztp-preparing-the-hub-cluster-for-ztp.adoc:103]: Good catch Michael, $ added.

**Pull request merged**